scripts/fix_transifex_resource_names.py

```diff
@@ -39,9 +39,23 @@ def is_dry_run():
     return '--dry-run' in sys.argv
 
 
+def get_transifex_project_slug():
+    """
+    Get Transifex project slug e.g. openedx-translations or openedx-translations-redwood.
+    """
+    slug = getenv('TRANSIFEX_PROJECT_SLUG')
+    if not slug:
+        raise RuntimeError(
+            'Error: Cannot determine Transifex project slug. Set `TRANSIFEX_PROJECT_SLUG` environment variable to '
+            '"openedx-translations" or "openedx-translations-redwood".'
```

Comment:
Do we want to mention `redwood` specifically here, or should this error be more generic (something like `openedx-translations-{release-name}`?

Reply:
That's a good point. I think we should use `release-name` everywhere in these docs, because otherwise we'll definitely forget to update this for every release.

You could provide a link to https://openedx.atlassian.net/wiki/spaces/OEPM/pages/4191191044/Open+edX+Releases+Homepage for the name of the current and next releases.

Reply:
Good note, done!